adapters/repos/db/aggregator/hybrid.go

```diff
@@ -51,10 +51,7 @@ func (a *Aggregator) bm25Objects(ctx context.Context, kw *searchparams.KeywordRa
 		cfg   = inverted.ConfigFromModel(class.InvertedIndexConfig)
 	)
 
-	objs, dists, err := inverted.NewBM25Searcher(cfg.BM25, a.store, s,
-		propertyspecific.Indices{}, a.classSearcher,
-		nil, a.propLengths, a.logger, a.shardVersion,
-	).BM25F(ctx, nil, a.params.ClassName, *a.params.ObjectLimit, *kw)
+	objs, dists, err := inverted.NewBM25Searcher(cfg.BM25, a.store, s, propertyspecific.Indices{}, a.classSearcher, nil, a.propLengths, a.logger, a.shardVersion, a.propertyIds).BM25F(ctx, nil, a.params.ClassName, *a.params.ObjectLimit, *kw)
```

Comment:
can we bring back the original formatting? bc there are no changes to this file except the formatting, no logic has changed here